payments-core/src/main/java/com/stripe/android/StripeIntentResult.kt

```diff
@@ -89,12 +89,12 @@ abstract class StripeIntentResult<out T : StripeIntent> internal constructor(
             StripeIntent.NextActionType.RedirectToUrl,
             StripeIntent.NextActionType.UseStripeSdk,
             StripeIntent.NextActionType.AlipayRedirect,
-            StripeIntent.NextActionType.BlikAuthorize,
             StripeIntent.NextActionType.WeChatPayRedirect,
             StripeIntent.NextActionType.CashAppRedirect,
             null -> {
                 false
             }
+            StripeIntent.NextActionType.BlikAuthorize,
```

Comment:
FYI, **this** and the change in `PaymentIntent` is the actual fix, everything else is to add the example.